etl/steps/data/garden/unicef/2023-06-16/diarrhea.meta.yml

```diff
@@ -0,0 +1,48 @@
+dataset:
+  title: Diarrhea treatment (UNICEF, 2022)
+  description: ''
+  licenses:
+  - name: Creative Commons Attribution-NonCommercial 3.0 IGO license
+    url: https://data.unicef.org/about-us/
+  sources:
+  - name: UNICEF (2022)
+    url: https://data.unicef.org/resources/dataset/diarrhoea/
+    source_data_url: https://data.unicef.org/wp-content/uploads/2018/06/Child-Health-Coverage-Database-May-2022.xlsx
+    date_accessed: '2023-06-16'
+    publication_date: '2022-05-01'
+    publication_year: 2022
+    published_by: Several health and multiple indicator cluster surveys compiled by
+      UNICEF
+tables:
```

Comment:
If the metadata above is exactly the same as meadow you can delete it and you avoid to update it multiple times in the future

Reply:
Ah thanks!